src/databases/keyvalue-indexed.js

```diff
@@ -21,6 +21,7 @@ import KeyValue from './keyvalue.js'
 import LevelStorage from '../storage/level.js'
 import pathJoin from '../utils/path-join.js'
 
+const type = 'keyvalueindexed'
```

Comment:
This should be the same as KeyValue.type as we want keyvalue-indexed to handle all keyvalue databases.

Reply:
So does this mean we shouldn't register the [keyvalue-indexed database here](https://github.com/orbitdb/orbitdb/blob/a0fc91d67497723600c80742042c0608c7d17ddb/src/databases/index.js#L62) because the implementer should be able to remove the keyvalue database and replace with the keyvalue-indexed?

Reply:
Correct. The registration effects which database orbitdb uses by default when opening a db (or creating one). If the user wants to open a db of type `keyvalue` they can open it without registering the database type and passing it as a param to the open() fn. If KVIndexed is registered, then it’ll be used (instead of KeyValue) as the default when opening a db of type `keyvalue`.